<!--
   Author: Steven Kordell, Worcester Polytechnic Institute
   Author: Russell Toris, Worcester Polytechnic Institute
  Version: July 24, 2014
-->

<launch>
  <arg name="linear_throttle_factor" value="1.0" />
  <arg name="angular_throttle_factor" value="1.0" />

  <node pkg="joy" type="joy_node" name="joy_node">
    <param name="deadzone" type="double" value="0.033" />
    <param name="autorepeat_rate" type="double" value="30" />
    <param name="coalesce_interval" type="double" value="0.033" />
  </node>
  <node pkg="carl_teleop" type="carl_joy_teleop" name="carl_joy_telep">
    <param name="linear_throttle_factor" type="double" value="$(arg linear_throttle_factor)" />
    <param name="angular_throttle_factor" type="double" value="$(arg angular_throttle_factor)" />
  </node>
</launch>
